src/modules/github/github.api.js

```diff
@@ -1,17 +1,56 @@
-class GitHubAPI {
-    url = 'https://api.github.com/';
+import { loadAllReposAction, searchReposAction } from "./github.actions";
```

Comment:
Raczej bym zrobił w drugą stronę tj. do akcji importował api i to tam tworzył `thunka`.
Tutaj tylko definiujesz zapytania do API.